factors.Rmd

```diff
@@ -88,239 +90,234 @@ f2 <- x1 %>% factor() %>% fct_inorder()
 f2
 ```
 
-If you ever need to access the set of valid levels directly, you can do so with `levels()`:
+Si alguna vez necesitas acceso directo al conjunto de niveles válidos, puedes hacerlo con `levels()` (del inglés _niveles_):
 
 ```{r}
 levels(f2)
 ```
 
-## General Social Survey
+## Encuesta Social General
 
-For the rest of this chapter, we're going to focus on `forcats::gss_cat`. It's a sample of data from the [General Social Survey](http://gss.norc.org), which is a long-running US survey conducted by the independent research organization NORC at the University of Chicago. The survey has thousands of questions, so in `gss_cat` I've selected a handful that will illustrate some common challenges you'll encounter when working with factors.
+Por el resto del capítulo, nos vamos a concentrar en `encuesta`. Ésta es la versión traducida al español de un conjunto de datos de ejemplo de [General Social Survey](http://gss.norc.org); ésta es una encuesta realizada en Estados Unidos desde hace mucho tiempo, conducida por la organización de investigación independiente llamada NORC, en la Universidad de Chicago. La encuesta tiene miles de preguntas, así que en el conjunto de datos he seleccionado aquellas que ilustran algunos de los desafíos comunes que encontrarás al trabajar con factores.
 
 ```{r}
-gss_cat
+library(datos)
+encuesta
 ```
 
-(Remember, since this dataset is provided by a package, you can get more information about the variables with `?gss_cat`.)
+(Recuerda, dado que este conjunto de datos está provisto por un paquete, puedes obtener más información de las variables con `?encuesta`.)
 
-When factors are stored in a tibble, you can't see their levels so easily. One way to see them is with `count()`:
+Cuando los factores están almacenados en un _tibble_ (se pronuncia /tibl/), no puedes ver sus niveles tán fácilmente. Una forma de verlos es con `count()` (del inglés _contar_):
 
 ```{r}
-gss_cat %>%
-  count(race)
+encuesta %>%
+  count(raza)
 ```
 
-Or with a bar chart:
+O con un gráfico de barras:
 
 ```{r}
-ggplot(gss_cat, aes(race)) +
+ggplot(encuesta, aes(raza)) +
   geom_bar()
 ```
 
-By default, ggplot2 will drop levels that don't have any values. You can force them to display with:
+Por defecto, __ggplot2__ retira los niveles que no tienen valores. Puedes forzarlos para que se visualicen con:
```

Comment:
Que tan comun is usar "por defecto", yo usualmente uso "De por si".  Pero puedo no estar en lo correcto :)

Reply:
@edgararuiz La verdad, que no lo sé. Quizás, tendríamos que llegar a una convención y agregarla al documento de lineamientos, para que todos traduzcamos del mismo modo.

Reply:
De acuerdo, este es algo tan pequeño que no deberia bloquear el merge de este PR, podemos acordar una convencion mas adelante

Reply:
“Por defecto” también se utiliza mucho en Chile.

Reply:
Yo uso "por defecto", pero no se cuan común es